js/libs/form.js

```diff
@@ -384,7 +388,11 @@ Fliplet.Widget.instance('form-builder', function(data) {
                 }
 
                 field.value = value;
-                $form.$forceUpdate();
+
+                // Wait for DOM to update, then force a refresh
+                setTimeout(function () {
```

Comment:
@squallstar Maybe use https://vuejs.org/v2/api/#Vue-nextTick instead of `setTimeout()`?

Reply:
@tonytlwu I've tried but it doesn't work.